packages/wdio-cli/src/constants.ts

```diff
@@ -165,15 +166,15 @@ export const QUESTIONNAIRE = [{
         'https',
         'http'
     ],
-    when: /* istanbul ignore next */ (answers) => {
+    when: /* istanbul ignore next */ (answers: any) => {
```

Comment:
Shouldn't answer be a string?

Reply:
Yes, but in order to be able to import package.json I had to make this file a `.ts` file. Someone who will rewrite this package can take care of actually typing these things

Reply:
Agree